docs/tutorial/01-hello/README.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Welcome to XOD
+---
+
+# #01. Welcome to XOD
+
+<div class="ui segment note">
+<span class="ui ribbon label">Note</span>
+This is a web-version of a tutorial chapter embedded right into the XOD IDE.
+To get a better learning experience we recommend to
+<a href="../install/">install the IDE</a>, launch it, and you’ll see the
+same tutorial there.
+</div>
+
+Welcome to XOD, dear Maker! Here, in XOD, we do not use text to code, but
+visual objects instead.
+
+![XOD window](./xod-window.png)
+
+## Patch board
+
+The large gray field with a box is your program. It’s called a *patch*. In this
+patch you can see one interesting object – the `led` node — a grey
+rectangle with two little green circles. You will be able to make complicated
+programs, using these rectangles.
+
+You can move nodes by dragging them. Position on the patch board does not
+affect your program, but can make it more readable.
+
+You can see a comment as well. Even though it is there, it does not affect the
+patch, because it is invisible to a machine. It’s only visible to you, a
+clever human. You can drag it to move too.
+
+## Project browser
+
+On the left side you will find a list with available projects. The list is
```

Comment:
"list with available projects" — it's not really true, cause we see only current project and projects in the library folder. Is this fact deliberately left out to simplify the learning?

Reply:
Well, yes. It’s like “numbers start with 1” at the first class in a school.

This is the first contact and we have to say something. “On the left side you will find a list with available… wut”?

Do you have an idea how to improve the sentence and not to complicate things at the same time? That would be nice!

Reply:
Maybe we could tell that user see his opened project and projects available from library. And tell more about libraries, terminal nodes and etc in the next tutorials. Like we do with pulse type, we introduce it and promise to tell more later :)

Reply:
OK. Changed it this way.

```diff
-On the left side you will find a list with available projects. The list is
-called a *Project Browser*. When you launch XOD for the first time the first
-item in the list is `welcome-to-xod`. This is a special tutorial project you
-can follow to learn XOD right inside the XOD IDE.
+On the left side you will find a list of patches grouped by a project or
+library name. The list is called a *Project Browser*. When you launch XOD for
+the first time the first item in the list is `welcome-to-xod`. This is a
+special tutorial project you can follow to learn XOD right inside the XOD IDE.
```